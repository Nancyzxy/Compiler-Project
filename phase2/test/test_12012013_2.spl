struct YY{
    float a[1];
};
int test_2_r01()
{
  int x4 = 3 + 1.6;
  struct YY yy;
  return yy.a[1];
}