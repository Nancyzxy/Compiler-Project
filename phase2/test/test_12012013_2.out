Error type 7 at Line 6: binary operation on non-number variables
Error type 8 at Line 8: incompatiable return type
